src/models/image.ts

```diff
@@ -6,11 +6,19 @@ import { EditionInfo } from './edition';
 
 export class IIIFImage {
     public url: string;
-    public manifest: any; // TODO: Create a Typescript interface for this
+    public manifest?: any; // TODO: Create a Typescript interface for this
```

Comment:
Try this perhaps?
```
interface Size {
    width: number;
    height: number;
}

interface Tile {
    width: number;
    scaleFactors: number[];
    height?: number;
}

interface Service {
    id: string;
    type: string;
    profile: string;
}

export default interface IiifInfoJson {
    '@context': string[];
    id: string | undefined;
    '@id': string | undefined;
    type: string;
    protocol: string;
    profile: string;
    width: number;
    height: number;
    maxWidth: number;
    maxHeight: number;
    maxArea: number;
    sizes: Size[];
    tiles: Tile[];
    rights: string;
    preferredFormats: string[];
    extraFormats: string[];
    extraQualities: string[];
    extraFeatures: string[];
    service: Service[];
}
```

Reply:
This is a good idea, I'll incorporate this into the code.